sw/host/opentitanlib/src/app/mod.rs

```diff
@@ -23,17 +23,62 @@ use std::collections::HashMap;
 use std::rc::Rc;
 use std::vec::Vec;
 
+const DEFAULT_TEMPLATE: &str = "[{elapsed_precise}] [{wide_bar}] {bytes}/{total_bytes} ({eta})";
+const STAGE_TEMPLATE: &str =
+    "{msg}: [{elapsed_precise}] [{wide_bar}] {bytes}/{total_bytes} ({eta})";
```

Comment:
This is more of a style thing/judgement call.  I prefer to scope consts to the entity that needs them.  For this one, it would be in the `StagedProgressBar` implementation:
```
impl StagedProgressBar {
    const STAGE_TEMPLATE: &str = ...;
    ...
}
```
}

Reply:
Done.